Modules/user.py

```diff
@@ -174,47 +134,25 @@ def nickname(self) -> str:
         return self._nickname
 
     @classmethod
-    async def from_whois(cls, bot: BasicClient, nickname: str) -> Optional['User']:
+    async def from_pydle(cls, bot: BasicClient, nickname: str) -> Optional['User']:
         """
-        Creates a user object from a WHOIS query
+        Returns a user object from pydle's backend
 
         Args:
-            bot(MechaClient): MechaClient instance
-            nickname(str):  nickname to look up
+            bot (BasicClient): Mechaclient instance
+            nickname (str): nickname of user
 
         Returns:
-            User: user object
-            None: no found user
-
-        Raises:
-            ValueError: somehow got an invalid WHOIS response
-            KeyError: malformed dict received from pydle
+            User: found user
+            None: user not found
         """
-        try:
-            data = await bot.whois(nickname)
-        except AttributeError:
-            # Pydle errors when the user is not found
-            return None
+        # fetch the user object from
+        data = bot.users.get(nickname.casefold(), None)
 
-        # anything less than 7 is not a valid whois reply
-        if len(data) < 7:
-            raise ValueError("Dict does not contain enough keys")
-
-        # full whois reply
-        else:
-            return cls(oper=data['oper'],
-                       idle=data['idle'],
-                       away=data['away'],
-                       away_message=data['away_message'],
-                       identified=data['identified'],
-                       secure=data['secure'],
-                       account=data['account'],
-                       nickname=nickname,
-                       username=data['username'],
-                       hostname=data['hostname'],
-                       realname=data['realname'],
-                       server=data['server'],
-                       server_info=data['server_info'])
+        # if we got a object back
+        # ((inb4 falsy check comes back to bite))
```

Comment:
I feel like the second one of these comments shouldn't be here ;)

Reply:
methinks its fine where it is, in (( out of context notation )) 😉

Reply:
im not seriously concerned, and im actually checking against a Empty dictionary or None. I just have a bit of a inherent distrust towards falsy checks 😅

Also, `(())` notation, while used in our drills, actually originates from elsewhere.


Anyways, if it bothers you this much il just remove the comment
```suggestion
        
```